src/fusor/harvester.py

```diff
@@ -334,14 +336,19 @@ def __init__(
             This parameter defaults to ``True``.
         :param local_cache_path: A filepath destination for the retrieved remote
             cache. This parameter defaults to LOCAL_CACHE_PATH from civicpy.
+        :param include_status: Whether to include accepted, submitted, and/or
+            rejected fusion variants from civicpy cache. By default, this is
+            set to None.
```

Comment:
Did you not want to have it so that if it's `None` then it will default to `accepted` (which is what it was before)? Either way, should update the docstring (if keeping as is, then this will allow all statuses)